pages/index.tsx

```
@@ -6,14 +6,21 @@ import { Nav } from "../components/Nav";
 import admins from "../lib/admins";
 import candidates from "../lib/candidates";
 import employers from "../lib/employers";
+import Error from "./_error";
 
 const IndexPage: React.SFC<{
-    candidateName: string;
+    candidateName: string | undefined;
     role: string;
     token: string;
     username: string;
     user_id: number;
 }> = ({ candidateName, role, token, username, user_id }) => {
+    if (!token) {
+        return <Error statusCode={404} />;
```

can we just throw an error?

throw new Error(/ * qsdkjqsd * /)